osbenchmark/worker_coordinator/runner.py

```diff
@@ -1272,6 +1277,20 @@ async def __call__(self, opensearch, params):
     def __repr__(self, *args, **kwargs):
         return "put-pipeline"
 
+class DeletePipeline(Runner):
+    @time_func
+    async def __call__(self, opensearch, params):
+        try:
+            await opensearch.ingest.delete_pipeline(id=mandatory(params, "id", self),
+                                                    master_timeout=params.get("master-timeout"),
+                                                    timeout=params.get("timeout"),
+                                                    )
+        except:
+            pass # no current pipeline
```

Comment:
Might be useful to still add a logging statement that states that there is no current pipeline to delete before swallowing the exception.

Reply:
Pylint also suggests that this is a bare except.
```
osbenchmark/worker_coordinator/runner.py:1288:8: W0702: No exception type(s) specified (bare-except)
osbenchmark/worker_coordinator/runner.py:2404:8: W0702: No exception type(s) specified (bare-except)
osbenchmark/worker_coordinator/runner.py:2444:8: W0702: No exception type(s) specified (bare-except)
```